components/api_version/src/api_v2.rs

```diff
@@ -147,7 +147,9 @@ impl APIVersion for APIV2 {
         debug_assert_eq!(Self::parse_key_mode(&user_key), KeyMode::Raw);
```

Comment:
It will be used in next PR #12128.
Please see [here](https://github.com/pingyu/tikv/blob/e922d6e2fea54969c336169669b104edf4af5fd9/src/storage/mod.rs#L1483).